runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/SamzaTimerInternalsFactory.java

```diff
@@ -444,6 +470,16 @@ private class SamzaTimerState {
                           new TimerKeyCoder<>(keyCoder, windowCoder),
                           VarLongCoder.of()));
 
+      this.timestampSortedProcessTimeTimerState =
+          (SamzaSetState<KeyedTimerData<K>>)
+              nonKeyedStateInternalsFactory
+                  .stateInternalsForKey(null)
+                  .state(
+                      StateNamespaces.global(),
+                      StateTags.set(
+                          timerStateId + "-pts",
```

Comment:
looks like we anyways don't have consistent naming scheme and can't change `-ts` to `-ets`. maybe be more descriptive instead of `pts` to help readability and debuggability.

Reply:
good point will name it to process-timestamp-sorted